virttest/env_process.py

```diff
@@ -303,6 +303,7 @@ def preprocess_vm(test, params, env, name):
                           migration_exec_cmd=params.get("migration_exec_cmd_dst"))
 
         # Update kernel param
+        serial_update = params.get_boolean("kernel_extra_params_serial_update")
```

Comment:
Recommend to name it `update_boot_option_serial_login`.

Reply:
The `kernel_extra_params` prefix is ​​used to maintain alignment with `kernel_extra_params_add` and `kernel_extra_params_remove`. `update_boot_option` is a method we invoked but not very suitable as a param name in `env_process`. How about keeping this one or using `kernel_extra_params_serial_login`?

Reply:
> The `kernel_extra_params` prefix is ​​used to maintain alignment with `kernel_extra_params_add` and `kernel_extra_params_remove`. `update_boot_option` is a method we invoked but not very suitable as a param name in `env_process`. How about keeping this one or using `kernel_extra_params_serial_login`?

Okay, make sense.